velox/exec/MergeJoin.h

```diff
@@ -355,8 +386,25 @@ class MergeJoin : public Operator {
 
   std::optional<LeftJoinTracker> leftJoinTracker_{std::nullopt};
 
+  // Indices buffer used by the the left and right dictionary outputs. All
+  // output from left and right share the same dictionaries.
+  BufferPtr leftIndices_;
+  BufferPtr rightIndices_;
+
+  vector_size_t* rawLeftIndices_;
+  vector_size_t* rawRightIndices_;
+
+  // Stores the current left and right vectors being used by the output
```

Comment:
Are these base vectors (original left and right vectors) or something else?

Reply:
the original ones.